cirq-core/cirq/ops/moment.py

```diff
@@ -483,6 +483,26 @@ def _commutes_(
 
         return True
 
+    def get_single_gate_from_moment(self) -> Optional['cirq.Operation']:
```

Comment:
Should this return a cirq.Gate instead?  If the gates match (but the qubits do not match, why does it pick the first operation?  Also, this is called get_single_gate_from_moment not from_single_op_from_moment

Reply:
Changed from Operation to Gate.